content/05.conclusion.md

```diff
@@ -1,19 +1,28 @@
-A benefit of analyzing bioRxiv text is that we can compare preprint and published versions to examine the influence of the publication process.
-Token-level analyses suggest that differences between corpora are driven by fields, while comparisons of preprints with their corresponding publication reveals differences in typesetting and supplementary materials.
+Preprints are typically linked with their published articles via bioRxiv manually establishing a link or authors self-reporting that their preprint has been published; however, gaps can occur as preprints change their appearance through multiple versions or authors do not notify bioRxiv. 
+We used the full text of documents to correct these missing annotations and our analysis provided a comprehensive understanding of the extent to which preprints are published.
+This correction resulted in a publication rate that is higher than previously estimated.
+Importantly, our analysis only accounts for papers that are published open access, so these results should be considered to raise the lower bound of the total preprint publication fraction; however, the true fraction is likely to be much higher.
+In any case, future work, especially that which aims to assess the fraction of preprints that are eventually published, should account for the possibility of missed annotations.
 
-Previous analyses have focused on article metadata, including which papers are heavily downloaded and discussed [@doi:10.7554/eLife.45133] and by which communities discuss them [@doi:10.1371/journal.pbio.3000860].
-We found that some preprints are high similar to published articles within the PMC open access corpus, and a detailed examination revealed that many preprints were published but not previously annotated.
-Using the full text of documents to correct these missing annotations provides a comprehensive understanding of the extent to which preprints are published and this correction resulted in a publication rate that is higher than previously estimated.
-Importantly, this only accounts for papers that are published open access, so our analysis should be considered to raise the lower bound but the truly published fraction is likely to be higher.
+Our work presents an early step towards understanding the process by which peer review alters life sciences papers.
+We analyzed and compared of the full text of preprints with their corresponding publication.
+Our analysis revealed differences in typesetting and supplementary materials, suggesting that the majority of peer review changes are modest at best. 
+Our results lay out the groundwork for future endeavors that aims to identify sentences and claims that are altered during the peer review process.
+Preprints in general take approximately 16 days on average to venture through the peer review process provided that modest changes are requested.

```

Comment:
I don't think your results show this. What's the average time on peer review? Why is a document distance of 1 the average?